scripts/enemy/EnemyFollowEnemyBehaviour.gd

```diff
@@ -0,0 +1,31 @@
+extends Node3D
+
+
+@onready var enemy: EnemyBase = $".."
```

Comment:
Is there a better way to reference this? 
Maybe you could export it and assign it directly in the scene
or make the scene unique to reference it

Reply:
This class should always have a EnemyBase script as the root